src/common/mfu_flist_copy.c

```diff
@@ -443,6 +447,37 @@ static int mfu_copy_xattrs(
     mfu_free(&list);
     list_bufsize = 0;
 
+    /* open the src */
+    int src_fd = open(src_path, O_RDONLY);
+    if (src_fd >= 0) {
+        /* open the dest */
+        int dest_fd = open(dest_path, O_RDONLY);
+        if (dest_fd >= 0) {
+            struct fsxattr attr;
+            /* get src's xattr */
+            rc = ioctl(src_fd, FS_IOC_FSGETXATTR, &attr);
+            if (rc == 0) {
+                /* set dest's xattr */
+                rc = ioctl(dest_fd, FS_IOC_FSSETXATTR, &attr);
```

Comment:
Is it safe to write all of `attr` like this?  For example, are fsx_nextents and fsx_extesize guaranteed not to change, if src_path is on one type of FS, like xfs, and dest_path is on another, like nfs?

Looking at https://man7.org/linux/man-pages/man2/ioctl_xfs_fsgetxattr.2.html it sounds to me like many of the flags and fields may be supported on some file systems but not others, or worse may be interpreted differently on one file system than on another.  e.g.

```
fsx_extsize is the preferred extent allocation size for data
blocks mapped to this file, in units of filesystem blocks.
```

Reply:
I would be more comfortable copying only specific flags or fields, such as `fsx_projid` and `fsx_xflags` bit `FS_XFLAG_PROJINHERIT`

Reply:
> Is it safe to write all of attr like this? For example, are fsx_nextents and fsx_extesize guaranteed not to change, if src_path is on one type of FS, like xfs, and dest_path is on another, like nfs?

What I mean to say is, "Should fsx_nextents and fsx_extesize be changed depending on the type of the destination file system?  Are there flags in fsx_xflags that must not be set for some file system types?"